app/helpers/data_lists_helper.rb

```diff
@@ -67,16 +67,11 @@ def dl_linked_version(description)
     end
     return new_words.join(" ")
   end
-  
-end
 
+  def make_indentation(n)
+    ## HTML string below needs to keep internal quotes as single, because it gets embedded in JSON
```

Comment:
It would be a bit more idiomatic for us to return something like `{"intent": 1}` as our JSON, then use that number to create the HTML in the CoffeeScript file.